inyoka_theme_default/static/style/inyoka/inyoka.less

```diff
@@ -231,6 +236,12 @@ input[type="text"], input[type="email"], input[type="password"], select {
   .form-control-focus();
 }
 
+table {
```

Comment:
this came also from the cherry-pick
https://github.com/MAH1987/theme-default/commit/24f977dfd02bcca7a561668f6502db46e4466823